core/controllers/acl_decorators.py

```diff
@@ -2919,6 +2919,23 @@ def test_can_edit(
             raise self.NotFoundException
         if role_services.ACTION_EDIT_ANY_QUESTION in self.user.actions:
             return handler(self, question_id, **kwargs)
+        if role_services.ACTION_EDIT_OWNED_QUESTION in self.user.actions:
+            skills = question_services.get_skills_linked_to_question(question_id)
+            if not skills:
+                raise self.UnauthorizedUserException(
+                    'You do not have credentials to edit this question.')
+            skill_ids = [skill.id for skill in skills]
```

Comment:
If you make this a set, queries will be quicker.

Reply:
Fixed this